answers.txt

```diff
@@ -4,4 +4,6 @@
 
 3. By having different branches we can have multiple persons working on the same project and when they are done they can just push it to the master branch, by this we can avoid conflicts of multiple copies of the same file
```

Comment:
@krishna-gurram I can only see 3 and 4 questions. Can you please answer 1 and 2 questions as well

Reply:
Yes I'll do that